### PR TITLE
Update example for menu payload

### PR DESCRIPTION
**Pull request by CikiMomogi**
https://github.com/grammyjs/website/issues/369#issuecomment-1159617425

### CODE REVIEW

File: site/docs/plugins/menu.md [hunk 1]
```diff
@@ -310,23 +310,31 @@ This is useful because it lets you store a little bit of data in a menu.
...
+      // Give the user 5 seconds to undo.
+      const text = Date.now() - Number(ctx.match) < 5000
+        ? "The operation was canceled successfully."
+        : "Too late. Your cat videos have already gone viral on the internet.";
+      ctx.reply(text);
+    },
   );
 
 bot.use(menu);
 bot.command("menu", async (ctx) => {
```
Comment (KnorpelSenf) [inline site/docs/plugins/menu.md:334]:
```suggestion
bot.command("publish", async (ctx) => {
```

File: site/docs/plugins/menu.md [hunk 2]
```diff
@@ -310,23 +310,31 @@ This is useful because it lets you store a little bit of data in a menu.
...
   .text(
-    { text: "I know my creator", payload: generatePayload },
-    (ctx) => ctx.reply(`I was created by ${ctx.match}!`),
+    { text: "ABORT!", payload: generatePayload },
+    (ctx) => {
+      // Give the user 5 seconds to undo.
+      const text = Date.now() - Number(ctx.match) < 5000
+        ? "The operation was canceled successfully."
+        : "Too late. Your cat videos have already gone viral on the internet.";
+      ctx.reply(text);
```
Comment (KnorpelSenf) [inline site/docs/plugins/menu.md:329]:
```suggestion
      await ctx.reply(text);
```

File: site/docs/plugins/menu.md [hunk 3]
```diff
@@ -310,23 +310,31 @@ This is useful because it lets you store a little bit of data in a menu.
...
+  return Date.now().toString();
 }
 
-const menu = new Menu("store-author-name-in-payload")
+const menu = new Menu("store-current-time-in-payload")
   .text(
-    { text: "I know my creator", payload: generatePayload },
-    (ctx) => ctx.reply(`I was created by ${ctx.match}!`),
+    { text: "ABORT!", payload: generatePayload },
+    (ctx) => {
```
Comment (KnorpelSenf) [inline site/docs/plugins/menu.md:324]:
```suggestion
    async (ctx) => {
```

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Review by KnorpelSenf — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]

**Comment by habemuscode** [comment]
As talked with @KnorpelSenf and @roj1512 we will start with a new workflow to manage external contributions.

@CikiMomogi Can you change the base branch to `grammyjs:menu-payload`? We will add translations in other PR managed internally

**Comment by KnorpelSenf** [comment]
We first have to create a branch with the correct name in the main repo before the base branch can be changed. I created `grammyjs:menu-payload` now and adjusted the base branch of this contribution.

**Pull request merged**